client/evm-tracing/src/formatters/blockscout.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019-2022 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
```

Comment:
License files should be updated, it's GPL so please replace with Webb Technologis Inc.